scss/_title-bars.scss

```diff
@@ -0,0 +1,51 @@
+.title-bar {
+  // scss-docs-start title-bar-css-vars
+  --#{$boosted-prefix}title-bar-padding-y: #{$title-bar-padding-y};
+  --#{$boosted-prefix}title-bar-font-size: #{$h2-font-size};
+  --#{$boosted-prefix}title-bar-letter-spacing: #{$h2-spacing};
+  --#{$boosted-prefix}title-bar-border-width: #{$title-bar-border-width};
+  --#{$boosted-prefix}title-bar-border-color: #{$gray-500};
+  --#{$boosted-prefix}title-bar-image-ratio: #{$title-bar-image-ratio};
+  // scss-docs-end title-bar-css-vars
+
+  border-bottom: var(--#{$boosted-prefix}title-bar-border-width) solid var(--#{$boosted-prefix}title-bar-border-color);
+
+  @include media-breakpoint-up(md) {
```

Comment:
Apparently in the DSM we have to following sizes for the header: 60px 50px 30px. It looks like [`.display-1`](https://boosted.orange.com/docs/5.1/content/typography/#display-headings). Can't we use this class instead?

Reply:
OK I understand, the breakpoints are different between the DSM for this component and our breakpoints in Boosted regarding headers... Let's add `.display-1` in this PR and fix the behavior in the CSS, and then create an issue to align the DSM and Boosted for the headers.

Reply:
Just a reminder for the one that will create the issue, the actual breakpoints for `display-*` are `sm` and `lg`. Here, we need `md` and `xl`.